.github/workflows/plugin_upstream_testing_base.yml

```diff
@@ -41,14 +42,77 @@ jobs:
           builder: "${{ steps.buildx.outputs.name }}"
           context: "./"
           push: false
-          load: true
```

Comment:
What's this change do?

Reply:
> [Load](https://docs.docker.com/engine/reference/commandline/buildx_build/#load) is a shorthand for --output=type=docker (default false)

We're already setting output explicitly so we don't need this line